src/lib/components/chat/ChatMessage.svelte

```diff
@@ -59,20 +60,26 @@
 	let pendingTimeout: ReturnType<typeof setTimeout>;
 
 	const renderer = new marked.Renderer();
-
 	// For code blocks with simple backticks
 	renderer.codespan = (code) => {
 		// Unsanitize double-sanitized code
 		return `<code>${code.replaceAll("&amp;", "&")}</code>`;
 	};
 
 	const options: marked.MarkedOptions = {
-		...marked.getDefaults(),
+		// ...marked.getDefaults(), // Breaks katex
```

Comment:
Do you know what breaks? We'll probably want to keep the same options as before.https://github.com/markedjs/marked/blob/2ec4c00966844abc86c4d5f6106d71ba7b8b9f82/src/defaults.ts

Reply:
I will investigate, looking at your link this is likely `extensions: null,`. We may add the markedKatex extension directly in the config or just get rid of the .getDefaults and manually import the same config

Reply:
Yes extensions is breaking markedKatex. One workaround is this:

```typescript
// eslint-disable-next-line @typescript-eslint/no-unused-vars
const { extensions, ...defaults } = marked.getDefaults() as marked.MarkedOptions & {
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    extensions: any;
};
const options: marked.MarkedOptions = {
    ...defaults,
    gfm: true,
    breaks: true,
    renderer,
};
```

I did add some typing and eslint-disable